src/overlays/effects/ovl_Effect_Ss_Kakera/z_eff_ss_kakera.c

```diff
@@ -8,50 +8,349 @@
 
 #define PARAMS ((EffectSsKakeraInitParams*)initParamsx)
 
-s32 EffectSsKakera_Init(GlobalContext* globalCtx, u32 index, EffectSs* this, void* initParamsx);
+u32 EffectSsKakera_Init(GlobalContext* globalCtx, u32 index, EffectSs* this, void* initParamsx);
 void EffectSsKakera_Update(GlobalContext* globalCtx, u32 index, EffectSs* this);
 void EffectSsKakera_Draw(GlobalContext* globalCtx, u32 index, EffectSs* this);
 
-#if 0
+void func_8097E130(EffectSs* this, GlobalContext* globalCtx);
+f32 func_8097E368(f32, s32);
+f32 func_8097E384(f32, s32);
+f32 func_8097E3C0(f32, s32);
+f32 func_8097E400(f32, s32);
+
 const EffectSsInit Effect_Ss_Kakera_InitVars = {
     EFFECT_SS_KAKERA,
     EffectSsKakera_Init,
 };
+KakeraColorStruct D_8097EAD8[] = { { 0, { 255, 255, 255 } }, { 0, { 235, 170, 130 } }, { 1, { 210, 190, 170 } } };
+f32 D_8097EAE4[] = { 1.0f, 100.0f, 40.0f, 5.0f, 100.0f, 40.0f, 5.0f, 100.0f, 40.0f, 5.0f };
+f32 D_8097EB0C[] = { 0.05f, 1.0f };
+f32 D_8097EB14[] = { 4.0f, 0.1f, 0.3f, 0.9f, -0.1f, -0.3f, -0.9f };
+f32 D_8097EB30[] = { 0.1f, 1.0f, 6.0f };
+f32 (*D_8097EB3C[])(f32, s32) = { func_8097E368, func_8097E384, func_8097E384, func_8097E384, func_8097E3C0,
+                                  func_8097E3C0, func_8097E3C0, func_8097E400, func_8097E400, func_8097E400 };
+f32 D_8097EB64[] = { 10.0f, 20.0f, 40.0f };
```

Comment:
On a similar note, I think quite a lot of this can be in-function static. Only the function array is a bit large, but that's very likely what it was given it's only used once. That would enable getting rid of the forward declarations, too.